src/openfermion/resource_estimates/pbc/README.md

```diff
@@ -0,0 +1,92 @@
+3.370137329, 3.370137329, 0.000000000'''
+cell.unit = 'B'
+cell.verbose = 0
+cell.build()
+
+kmesh = [1, 1, 3]
+kpts = cell.make_kpts(kmesh)
+nkpts = len(kpts)
+mf = scf.KRHF(cell, kpts).rs_density_fit()
+    scf_dict = chkfile.load(_TEST_CHK, "scf")
+    mf.__dict__.update(scf_dict)
+    mf.with_df._cderi = _TEST_CHK
+    dm0 = mf.make_rdm1()
+mf.with_df.mesh = mf.cell.mesh
+mf.kernel()
+```
+
+then resource estimates for the SF, DF, and THC factorization schemes can be generated a range of cutoffs. For example:
+
+```python
+from openfermion.resource_estimates.pbc import sf
+
+costs = sf.generate_costing_table(mf, name='carbon_diamond', naux_cutoffs=[20,25,30,35,40,45,50])
+print(costs.to_string(index=False))
+```
+will generate a `pandas.DataFrame` of resource estimates for the single factorization Hamiltonian (`sf`) and MP2 correlation energies for the range of auxiliary dimension (`naux_cutoffs`).
+
+
+Note that the automated costing computes the MP2 correlation energy error as a reference point for monitoring the convergence of the factorization with respect to sparsity or the size of auxiliary dimension. MP2 may be a poor model chemistry depending on the system, changing the option `energy_method = "CCSD"` will use CCSD instead, but this may become too expensive as the system size grows.
+
+The philosophy is that all costing methods are captured in the namespace related to the type of factorization. So if one wanted to repeat the costing for DF or THC factorizations, one could do 
+
+```python
+from openfermion.resource_estimates.pbc import df, thc
+
+# We need to specify eigenvalue threshold for second factorization.
+df_cutoffs = [1e-2, 5e-3, 1e-3, 5e-4, 1e-4, 5e-5, 1e-5]
+df_table = df.generate_costing_table(
+    mf,
+    name='carbon-diamond',
+    cutoffs=df_cutoffs)
+
+# Specify THC rank parameter we wish to scan over.
+# Note THC dimension M = thc_rank_param * N, N = number of spin orbitals in the
+# unit cell. 
+thc_rank_params = [2, 4, 6]
+# if you want to save each THC result to a file, you can set 'save_thc' to True
+thc_table = thc.generate_costing_table(mf, name='carbon-diamond', thc_rank_params=thc_rank_params) 
+```
+
+More fine-grained control is given by subroutines that compute the factorization, the lambda values, and the cost estimates.
+Further details are provided in a [tutorial](./notebooks/resource_estimates.ipynb). Note the THC factorization is more involved and we refer the reader to [thc-tutorial](./notebooks/isdf.ipynb) for further details.
+
+Similar to the case of molecular resource estimation, we do not wish to burden all OpenFermion users with these dependencies, and testing with GitHub workflows is disabled. Currently we only check if pyscf is available. If it is then pytest will pick up the pbc module and run the tests. Note the tests can be quite slow due to the cost associated with building the integrals.
+
+## Requirements
+
+```
+pyscf
+jax
+jaxlib
+ase
+pandas
+```
```

Comment:
can you document just the additional requirements and where in `dev_tools/....` the official list can be found